### PR TITLE
(chore) - pkg.json metadata fields

### PR DESCRIPTION
**Pull request by JoviDeCroock**
Fixes: https://github.com/FormidableLabs/urql/issues/266

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,11 +2,21 @@
...
   "description": "A highly customizable and versatile GraphQL client for React",
+  "private": false,
+  "repository": "https://www.github.com/FormidableLabs/urql",
+  "bugs": "https://github.com/FormidableLabs/urql/issues",
   "main": "dist/urql.js",
   "module": "dist/urql.es.js",
   "types": "dist/types/index.d.ts",
   "source": "src/index.ts",
   "sideEffects": false,
+  "keywords": [
```
Comment (kitten) [inline package.json]: Let's add `graphql client` as one word, `state management` as two, and add `exchanges` :+1:

File: package.json [hunk 2]
```diff
@@ -2,11 +2,21 @@
   "name": "urql",
   "version": "1.0.5",
   "description": "A highly customizable and versatile GraphQL client for React",
+  "private": false,
+  "repository": "https://www.github.com/FormidableLabs/urql",
+  "bugs": "https://github.com/FormidableLabs/urql/issues",
```
Comment (kitten) [inline package.json]: I think this is typically in object of the form:

```js
  "bugs": {
    "url": "https://github.com/FormidableLabs/urql/issues"
  },
```

File: package.json [hunk 3]
```diff
@@ -2,11 +2,21 @@
   "name": "urql",
   "version": "1.0.5",
   "description": "A highly customizable and versatile GraphQL client for React",
+  "private": false,
```
Comment (kitten) [inline package.json]: This isn't needed if it's `false`

File: package.json [hunk 4]
```diff
@@ -2,11 +2,21 @@
   "name": "urql",
   "version": "1.0.5",
   "description": "A highly customizable and versatile GraphQL client for React",
```
Comment (kitten) [inline package.json]: Let's also add this :tada:

```suggestion
  "homepage": "https://formidable.com/open-source/urql",
```

File: package.json (hunk 1 above)
Comment (JoviDeCroock) [reply package.json]: I think I got it right, not entirely sure got confused with one word and two. So I made graphql -> graphql client state-management -> state management and added exchanges

**Pull request merged**